### DISCUSSION TITLE
change fingerprint

### DISCUSSION THREAD

**Discussion by idabough** [seleniumbase/SeleniumBase#3365] [General]
hi, I'm trying to build a antidetect browser using SeleniumBase.
i know i can use agent to change the userAgent of my page, but i neet to go advance now, if any chance to i can chage to fake those settinge (Platform, HardwareConcurency, Screen resolution, Fonts,WebGL ...) please help me

**Comment by mdmintz** [comment]
SeleniumBase gives you a browser that appears as a regular human-controlled web browser. It doesn't change fingerprints.

**Reply by idabough** [reply]
okay sir thank you, i have another question when i use proxy i get this notification from browser any idea why please![IMG_20241225_151803_684.jpg](https://github.com/user-attachments/assets/ab37fe6e-7346-44b3-952a-1367d92010c2)

**Reply by mdmintz** [reply]
It's safe to ignore that proxy warning. Hiding it would make the browser detectable.

Also, when I mentioned that `SeleniumBase gives you a browser that appears as a regular human-controlled web browser.`, I meant that specifically for **UC Mode** / [CDP Mode](https://github.com/seleniumbase/SeleniumBase/blob/master/examples/cdp_mode/ReadMe.md), in case there was any confusion.

**Reply by idabough** [reply]
ok thank mdmintz i will ignore it. yes i'm using UC mode but i didn't understand the CDP mode and what is the difference between UC and CDP mode. can i have small explanation for that and thank u  the time

**Reply by mdmintz** [reply]
CDP Mode lets you interact with the browser (via the cdp-driver) while the webdriver is disconnected from the browser in UC Mode. With regular UC Mode, you only get real stealth when you can't interact with the browser (which is fine for basic bypasses of Cloudflare CAPTCHAs that appear on page-loads, but not good enough for most other things).

**Reply by bshsbr** [reply]
> SeleniumBase gives you a browser that appears as a regular human-controlled web browser. It doesn't change fingerprints.

A big shoutout to Michael Mintz for creating the amazing SeleniumBase! It has made my work so much easier with all the fantastic features packed inside. SeleniumBase truly transformed Selenium, pushing it to grow and evolve in ways comparable to Playwright.

Have you heard about the BrowserForge project? I’m currently working on a modular project built on top of SeleniumBase. For now, I’m using BrowserForge to change the user agent and some other browser settings. It could be exciting to explore integrating BrowserForge into SeleniumBase in the future—it seems like it would be a great addition!

**Comment by 34892002** [comment]
@bshsbr Maybe you can try Camoufox, which has already been integrated with BrowserForge.
